### PR TITLE
Class features part 2

### PR DESCRIPTION
**Pull request by josephpatrickgreen**
## Amazon: Strong Throw
Amazons now have a stronger and longer distance throw, inspired by their javelin throwing.

## Bard: College Selection
Five Bard colleges have been added which you can choose on spawn:
-College of Lore: +2 reading, +1 music
-College of Eloquence: add trait noble
-College of Spirits: Add trait spiritual, add trait soul examine; +2 medicine
-College of Swords: +1 swords +1 knives
-College of Whispers: +2 sneaking, +2 lockpicking, give lockpicks

## Monk: Martial Artist
The monk gains the martial artist trait. This for now just makes their kicks stronger and able to target the head.

## Thug: Tavern Brawler
Tavern brawler has been reworked to actually do something. Select non-martial items now have the "improvised" variable on them which a tavern brawler does 1.5x damage with. Eg. cups, bowls, steel pans

## Fighter: Fighting Styles, Second Wind, Duelist
Renamed "warrior" baseclass to "fighter"

moved rogue's "duelist" subclass to fighter and debuffed it because of the fighting styles it now gets to select.

Moved duelist to fighter and 
Fighters can pick 2 fighting styles from the following:
-Dirty Fighting: Give nutcracker
-Shield Master: +2 shields
-Tavern Brawler: Give tavern brawler
-Unarmed Fighting: Give pugilist, +2 unarmed
-Thrown Weapon Fighting: Give strong throw, +2 perception 
-Mariner: +3 climbing, +3 swimming 
-Martial Artist: Give martial artist, +1 unarmed 
-Weapon master: +1 swords, polearms, maces and axes 
-Wrestling: Give strong grabber, +2 wrestling

Added second wind, a 10 minute cooldown spell that completely fills your stamina.

## Rogue: Thief, Swashbuckler
Renamed rogue's "rogue" archetype to "thief"
Ported swashbuckler from ratkeep

### CODE REVIEW

File: code/game/objects/items/rogueitems/natural/stones.dm [hunk 1]
```diff
@@ -141,6 +141,7 @@ GLOBAL_LIST_INIT(stone_personality_descs, list(
 	destroy_sound = 'sound/foley/hit_rock.ogg'
 	grind_results = list(/datum/reagent/consumable/sodiumchloride = 15)
 	var/magicstone = FALSE
+	improvised = TRUE
```
[the remark this answers is not in the document]
Comment (josephpatrickgreen) [reply code/game/objects/items/rogueitems/natural/stones.dm:144]: good point, also i dont think you can attack with rocks.

**Comment by josephpatrickgreen** [comment]
please wait. I am adding more...

**Comment by josephpatrickgreen** [comment]
Still not done.

File: code/game/objects/items/rogueitems/natural/stones.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (josephpatrickgreen) [reply code/game/objects/items/rogueitems/natural/stones.dm]: rock gaming has been removed

**Comment by Drayco84** [comment]
Give Amazons the improved dodge skill or whatever it is. They start with chain bikini armor and are obviously supposed to be a mobile fighter. Also, let them start with a chainskirt as well for some leg protection that still fits the "sexy warrior" theme.
I don't think an improved throw is gonna be all that great for them unless quivers can hold javelins or throwing spears, in which case they should start with those too.
A lantern would also be nice, but then I'll say that about every single character... I'd settle for an iron torch, though.

**Comment by josephpatrickgreen** [comment]
im done for now

**Pull request merged**